Sources/AST/ASTDumper.swift

```diff
@@ -87,243 +93,264 @@ class ASTDumper<StreamType: ColoredStream>: ASTTransformer {
     }
   }
-  override func visitInfixOperatorExpr(_ expr: InfixOperatorExpr) -> Result {
+  public override func visitInfixOperatorExpr(_ expr: InfixOperatorExpr) {
     printNode(expr) {
       super.visitInfixOperatorExpr(expr)
     }
   }
-  override func visitPrefixOperatorExpr(_ expr: PrefixOperatorExpr) -> Result {
+  public override func visitPrefixOperatorExpr(_ expr: PrefixOperatorExpr) {
     printNode(expr) {
       super.visitPrefixOperatorExpr(expr)
     }
   }
-  override func visitPropertyRefExpr(_ expr: PropertyRefExpr) -> Result {
+  public override func visitPropertyRefExpr(_ expr: PropertyRefExpr) {
     printNode(expr) {
       super.visitPropertyRefExpr(expr)
     }
   }
-  override func visitNilExpr(_ expr: NilExpr) {
+  public override func visitNilExpr(_ expr: NilExpr) {
     printNode(expr) {
       super.visitNilExpr(expr)
     }
   }
-  override func visitPoundFunctionExpr(_ expr: PoundFunctionExpr) {
+  public override func visitPoundFunctionExpr(_ expr: PoundFunctionExpr) {
     printNode(expr) {
       super.visitPoundFunctionExpr(expr)
     }
   }
-  override func visitArrayExpr(_ expr: ArrayExpr) {
+  public override func visitArrayExpr(_ expr: ArrayExpr) {
     printNode(expr) {
       super.visitArrayExpr(expr)
     }
   }
-  override func visitSizeofExpr(_ expr: SizeofExpr) {
+  public override func visitSizeofExpr(_ expr: SizeofExpr) {
     printNode(expr) {
       super.visitSizeofExpr(expr)
     }
   }
-  override func visitOperatorDecl(_ decl: OperatorDecl) {
+  public override func visitOperatorDecl(_ decl: OperatorDecl) {
     printNode(decl) {
       super.visitOperatorDecl(decl)
     }
   }
-  override func visitTupleExpr(_ expr: TupleExpr) {
+  public override func visitTupleExpr(_ expr: TupleExpr) {
     printNode(expr) {
       super.visitTupleExpr(expr)
     }
   }
-  override func visitTupleFieldLookupExpr(_ expr: TupleFieldLookupExpr) {
+  public override func visitTupleFieldLookupExpr(_ expr: TupleFieldLookupExpr) {
     printNode(expr) {
       super.visitTupleFieldLookupExpr(expr)
     }
   }
-  override func visitParenExpr(_ expr: ParenExpr) -> Result {
+  public override func visitParenExpr(_ expr: ParenExpr) {
     printNode(expr) {
       super.visitParenExpr(expr)
     }
   }
-  override func visitPoundDiagnosticStmt(_ stmt: PoundDiagnosticStmt) -> () {
+  public override func visitPoundDiagnosticStmt(_ stmt: PoundDiagnosticStmt) {
     printNode(stmt) {
       super.visitPoundDiagnosticStmt(stmt)
     }
   }
-  override func visitIsExpr(_ expr: IsExpr) -> Void {
+  public override func visitIsExpr(_ expr: IsExpr) -> Void {
     printNode(expr) {
       super.visitIsExpr(expr)
     }
   }
-  override func visitCoercionExpr(_ expr: CoercionExpr) -> Void {
+  public override func visitCoercionExpr(_ expr: CoercionExpr) -> Void {
     printNode(expr) {
       super.visitCoercionExpr(expr)
     }
   }
 }
+
+extension String {
+    func splitCapitals() -> [String] {
```

Comment:
this is very not-unicode aware. Couldn't you instead split by characters, and check to see if `c.uppercased == c`?